src/library/scala/collection/immutable/StrictOptimizedSeqOps.scala

```diff
@@ -77,4 +77,22 @@ trait StrictOptimizedSeqOps[+A, +CC[_], +C]
     b.result()
   }
 
+  override def sorted[B >: A](implicit ord: Ordering[B]): C = {
```

Comment:
Actually, do we need this change in 2.13.0 to prepare for this PR in 2.13.1?

Reply:
Oh, yes, adding the delegate to `super.sorted` would be very nice.

Reply:
if you could submit the minimal version of this PR with that override, we can probably squeeze it in for RC3 -- I think the rest can then come in 2.13.1

Reply:
(Tbh, while nice, I don't think we should hold up RC3 over this. I'll keep y'all posted about its timing, because we did hit a snag with scala.js in #8083, and may need to delay the RC to give Sébastien some breathing room.)

Reply:
Sure thing @adriaanm 😄 totally up to you https://github.com/scala/scala/pull/8108